core/templates/pages/skill-editor-page/navbar/skill-editor-navbar.directive.ts

```diff
@@ -35,18 +35,13 @@ angular.module('oppia').directive('skillEditorNavbar', [
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/skill-editor-page/navbar/skill-editor-navbar.directive.html'),
       controller: [
-        '$scope', '$uibModal', '$window', 'AlertsService',
-        'UndoRedoService', 'SkillEditorStateService',
-        'SkillEditorRoutingService',
-        'EVENT_SKILL_INITIALIZED', 'EVENT_SKILL_REINITIALIZED',
-        'EVENT_UNDO_REDO_SERVICE_CHANGE_APPLIED',
+        '$scope', '$uibModal', 'AlertsService', 'UndoRedoService',
+        'SkillEditorStateService', 'SkillEditorRoutingService',
         function(
-            $scope, $uibModal, $window, AlertsService,
-            UndoRedoService, SkillEditorStateService,
-            SkillEditorRoutingService,
-            EVENT_SKILL_INITIALIZED, EVENT_SKILL_REINITIALIZED,
-            EVENT_UNDO_REDO_SERVICE_CHANGE_APPLIED) {
+            $scope, $uibModal, AlertsService, UndoRedoService,
+            SkillEditorStateService, SkillEditorRoutingService) {
           var ctrl = this;
+          ctrl.subscriptions = [];
```

Comment:
Is this used?

Reply:
No, It has to be removed, this was the old way of keeping track of subscriptions.

Reply:
Removed!